### ISSUE TITLE
Flutter web support?

### ISSUE THREAD

**Issue by mirkokiefer** [shekohex/flutterust#8]
Would it be possible to produce Rust targets that work with Flutter web?

**Comment by shekohex** [comment]
Yes, in theory that's possible .. but it would require that your Rust code compiled to WASM and then you could load it in Flutter using dart wasm https://github.com/dart-lang/sdk/issues/37355

**Comment by iapicca** [comment]
> Yes, in theory that's possible .. but it would require that your Rust code compiled to WASM and then you could load it in Flutter using dart wasm [dart-lang/sdk#37355](https://github.com/dart-lang/sdk/issues/37355)

I hope it will e reality one day
https://github.com/flutter/flutter/issues/41062

**Comment by pratikpparikh** [comment]
@mirkokiefer @shekohex @iapicca just out curiosity. Can the web suppot be not enabled by add support for wasm at rust level?

**Comment by fzyzcjy** [comment]
Yes, Rust+Flutter Web works: https://github.com/fzyzcjy/flutter_rust_bridge
